test/unit/report_importer_test.rb

```diff
@@ -87,3 +89,9 @@ class ReportImporterTest < ActiveSupport::TestCase
     assert_equal host, db_host
   end
 end
+
+class TestReportImporter < ReportImporter
+  def report_status
+    {}
```

Comment:
`0`?  It's an integer field, I copied this into my own plugin and validation of new Report objects fails.  You can add an `assert_valid r` to one of the early tests to see the same.